.github/workflows/test.yml

```
@@ -30,3 +30,4 @@ jobs:
       - name: Nox build
         run: |
           python -m nox -s build
```

When you add a matrix per my comment above, the only thing left would be just using the value here
```suggestion
          python -m nox -s ${{ matrix.noxenv }}
```